src/app.jsx

```diff
@@ -333,6 +390,9 @@ var Pager = React.createClass({
         if (this.props.count == 0) {
             return <div />;
         }
+        if (!this.props.showresult) {
+            return <div />;
+        }
         var n_pages = Math.ceil(this.props.count / PAGE_SIZE);
```

Comment:
Could above 2 conditions be:
if (this.props.count == 0 or !this.props.showresult) {
    return <div />;
}